include/open62541/plugin/nodestore.h

```diff
@@ -606,6 +612,7 @@ typedef struct {
     UA_NodeHead head;
     UA_NODE_VARIABLEATTRIBUTES
     UA_Byte accessLevel;
+    UA_Byte userAccessLevel;
```

Comment:
The UserAccessLevel is computed dynamically for each session. Don't store it in the node.